src/goto-programs/structured_trace_util.cpp

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************\
+
+Module: Goto Programs
```

Comment:
Nit: Not goto programs module :palm_tree:

Reply:
What module would you prefer? It is in folder goto-programs?

Reply:
I think normally this just names the feature in the file, here it would be Structured Trace Utilities or something.

Reply:
This seems... pointless :stuck_out_tongue: I've checked the coding standards and it says it should just mention author, so I'll delete this line